src/templates/parts/actor-inventory.hbs

```diff
@@ -69,6 +69,9 @@
     {{#each section.items as |item iid|}}
     <li class="item flexrow {{section.css}}"
       data-item-id="{{#if section.editableName}}{{iid}}{{else}}{{item._id}}{{/if}}">
+
+      <div class="item-image" style="background-image: url({{item.img}})"></div>
```

Comment:
It would be better if this was not rendered `{{#unless` the appropriate game setting was set. Then you wouldn't have to worry about adding/removing the class to the element.